tests/cursor/cursor-destruct-001.phpt

```diff
@@ -25,22 +25,24 @@ function getNumOpenCursors(MongoDB\Driver\Manager $manager)
 
 $manager = new MongoDB\Driver\Manager(URI);
 
+$server = $manager->selectServer(new \MongoDB\Driver\ReadPreference('primary'));
```

Comment:
I assume this was required because you added a second mongos to the sharded cluster MO config? Should we add a one-line comment here to explain? Feel free to copy-paste it to other places you had to make the same test change.

Reply:
That is correct. All tests that relied on server selection in manager.c now select the server themselves to direct all future operations (`serverStatus` in the case of this test) to the same server.